From 75e8d551121b6dfbcc5148311e3cd1fcb8ae6593 Mon Sep 17 00:00:00 2001
From: auto3000 <auto3000@users.noreply.github.com>
Date: Sun, 11 Jun 2017 16:28:26 +0000
Subject: [PATCH] prebuilt: add .ttl files from prebuilts

TTL files are generated by tools compiled by Makefile. However,
the Makefile and these tools does not support cross compilation scheme.
This modification disable the generation of TTL file from Makefile.
---
 Makefile                            |   2 +-
 bin/Harmless.lv2/Harmless_dsp.ttl   | 108 ++++++++++++++++++++++++++++++
 bin/Harmless.lv2/manifest.ttl       |  10 +++
 bin/Larynx.lv2/Larynx_dsp.ttl       |  78 ++++++++++++++++++++++
 bin/Larynx.lv2/manifest.ttl         |  10 +++
 bin/Modulay.lv2/Modulay_dsp.ttl     | 114 +++++++++++++++++++++++++++++++
 bin/Modulay.lv2/manifest.ttl        |  10 +++
 bin/Pitchotto.lv2/Pitchotto_dsp.ttl | 115 ++++++++++++++++++++++++++++++++
 bin/Pitchotto.lv2/manifest.ttl      |  10 +++
 bin/Shiroverb.lv2/Shiroverb_dsp.ttl | 129 ++++++++++++++++++++++++++++++++++++
 bin/Shiroverb.lv2/manifest.ttl      |  10 +++
 plugins/Makefile.mk                 |   2 +-
 12 files changed, 596 insertions(+), 2 deletions(-)
 create mode 100644 bin/Harmless.lv2/Harmless_dsp.ttl
 create mode 100644 bin/Harmless.lv2/manifest.ttl
 create mode 100644 bin/Larynx.lv2/Larynx_dsp.ttl
 create mode 100644 bin/Larynx.lv2/manifest.ttl
 create mode 100644 bin/Modulay.lv2/Modulay_dsp.ttl
 create mode 100644 bin/Modulay.lv2/manifest.ttl
 create mode 100644 bin/Pitchotto.lv2/Pitchotto_dsp.ttl
 create mode 100644 bin/Pitchotto.lv2/manifest.ttl
 create mode 100644 bin/Shiroverb.lv2/Shiroverb_dsp.ttl
 create mode 100644 bin/Shiroverb.lv2/manifest.ttl

diff --git a/Makefile b/Makefile
index e8eaca3..11de58e 100644
--- a/Makefile
+++ b/Makefile
@@ -6,7 +6,7 @@
 
 include Makefile.mk
 
-all: plugins gen
+all: plugins 
 
 # --------------------------------------------------------------
 
diff --git a/bin/Harmless.lv2/Harmless_dsp.ttl b/bin/Harmless.lv2/Harmless_dsp.ttl
new file mode 100644
index 0000000..7c4f24e
--- /dev/null
+++ b/bin/Harmless.lv2/Harmless_dsp.ttl
@@ -0,0 +1,108 @@
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix mod:  <http://moddevices.com/ns/mod#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rsz:  <http://lv2plug.in/ns/ext/resize-port#> .
+@prefix unit: <http://lv2plug.in/ns/extensions/units#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/harmless>
+    a lv2:ModulatorPlugin, lv2:Plugin ;
+
+    lv2:extensionData <http://lv2plug.in/ns/ext/state#interface> ;
+
+    lv2:optionalFeature <http://lv2plug.in/ns/ext/buf-size#boundedBlockLength> ;
+
+    lv2:requiredFeature <http://lv2plug.in/ns/ext/options#options> ,
+                        <http://lv2plug.in/ns/ext/urid#map> ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:AudioPort ;
+        lv2:index 0 ;
+        lv2:symbol "lv2_audio_in_1" ;
+        lv2:name "Audio Input 1" ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:AudioPort ;
+        lv2:index 1 ;
+        lv2:symbol "lv2_audio_in_2" ;
+        lv2:name "Audio Input 2" ;
+    ] ;
+
+    lv2:port [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 2 ;
+        lv2:symbol "lv2_audio_out_1" ;
+        lv2:name "Audio Output 1" ;
+    ] ,
+    [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 3 ;
+        lv2:symbol "lv2_audio_out_2" ;
+        lv2:name "Audio Output 2" ;
+    ] ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 4 ;
+        lv2:name "rate" ;
+        lv2:symbol "rate" ;
+        lv2:default 4.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 20.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 5 ;
+        lv2:name "shape" ;
+        lv2:symbol "shape" ;
+        lv2:default 0.500000 ;
+        lv2:minimum 0.010000 ;
+        lv2:maximum 0.990000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 6 ;
+        lv2:name "tone" ;
+        lv2:symbol "tone" ;
+        lv2:default 6000.000000 ;
+        lv2:minimum 500.000000 ;
+        lv2:maximum 6000.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 7 ;
+        lv2:name "phase" ;
+        lv2:symbol "phase" ;
+        lv2:default 0.000000 ;
+        lv2:minimum -180.000000 ;
+        lv2:maximum 180.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 8 ;
+        lv2:name "depth" ;
+        lv2:symbol "depth" ;
+        lv2:default 100.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ;
+
+    rdfs:comment """
+Harmless is a waveshapeable harmonic tremolo with a stereo phase control.
+""" ;
+
+    mod:brand "SHIRO" ;
+    mod:label "Harmless" ;
+
+    doap:name "Harmless" ;
+    doap:license "ISC" ;
+
+    doap:maintainer [
+        foaf:name "DISTRHO" ;
+        foaf:homepage <https://github.com/DISTRHO/DPF-Max-Gen> ;
+    ] ;
+
+    lv2:microVersion 1 ;
+    lv2:minorVersion 0 .
+
diff --git a/bin/Harmless.lv2/manifest.ttl b/bin/Harmless.lv2/manifest.ttl
new file mode 100644
index 0000000..774e9c1
--- /dev/null
+++ b/bin/Harmless.lv2/manifest.ttl
@@ -0,0 +1,10 @@
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/harmless>
+    a lv2:Plugin ;
+    lv2:binary <Harmless_dsp.so> ;
+    rdfs:seeAlso <Harmless_dsp.ttl> ,
+                 <modgui.ttl> .
+
+
diff --git a/bin/Larynx.lv2/Larynx_dsp.ttl b/bin/Larynx.lv2/Larynx_dsp.ttl
new file mode 100644
index 0000000..a186934
--- /dev/null
+++ b/bin/Larynx.lv2/Larynx_dsp.ttl
@@ -0,0 +1,78 @@
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix mod:  <http://moddevices.com/ns/mod#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rsz:  <http://lv2plug.in/ns/ext/resize-port#> .
+@prefix unit: <http://lv2plug.in/ns/extensions/units#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/larynx>
+    a lv2:ModulatorPlugin, lv2:Plugin ;
+
+    lv2:extensionData <http://lv2plug.in/ns/ext/state#interface> ;
+
+    lv2:optionalFeature <http://lv2plug.in/ns/ext/buf-size#boundedBlockLength> ;
+
+    lv2:requiredFeature <http://lv2plug.in/ns/ext/options#options> ,
+                        <http://lv2plug.in/ns/ext/urid#map> ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:AudioPort ;
+        lv2:index 0 ;
+        lv2:symbol "lv2_audio_in_1" ;
+        lv2:name "Audio Input 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 1 ;
+        lv2:symbol "lv2_audio_out_1" ;
+        lv2:name "Audio Output 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 2 ;
+        lv2:name "tone" ;
+        lv2:symbol "tone" ;
+        lv2:default 6000.000000 ;
+        lv2:minimum 500.000000 ;
+        lv2:maximum 12000.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 3 ;
+        lv2:name "depth" ;
+        lv2:symbol "depth" ;
+        lv2:default 1.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 5.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 4 ;
+        lv2:name "rate" ;
+        lv2:symbol "rate" ;
+        lv2:default 5.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 10.000000 ;
+    ] ;
+
+    rdfs:comment """
+Larynx is a simple vibrato with a tone control.
+""" ;
+
+    mod:brand "SHIRO" ;
+    mod:label "Larynx" ;
+
+    doap:name "Larynx" ;
+    doap:license "ISC" ;
+
+    doap:maintainer [
+        foaf:name "DISTRHO" ;
+        foaf:homepage <https://github.com/DISTRHO/DPF-Max-Gen> ;
+    ] ;
+
+    lv2:microVersion 1 ;
+    lv2:minorVersion 0 .
+
diff --git a/bin/Larynx.lv2/manifest.ttl b/bin/Larynx.lv2/manifest.ttl
new file mode 100644
index 0000000..caca30e
--- /dev/null
+++ b/bin/Larynx.lv2/manifest.ttl
@@ -0,0 +1,10 @@
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/larynx>
+    a lv2:Plugin ;
+    lv2:binary <Larynx_dsp.so> ;
+    rdfs:seeAlso <Larynx_dsp.ttl> ,
+                 <modgui.ttl> .
+
+
diff --git a/bin/Modulay.lv2/Modulay_dsp.ttl b/bin/Modulay.lv2/Modulay_dsp.ttl
new file mode 100644
index 0000000..440f2b4
--- /dev/null
+++ b/bin/Modulay.lv2/Modulay_dsp.ttl
@@ -0,0 +1,114 @@
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix mod:  <http://moddevices.com/ns/mod#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rsz:  <http://lv2plug.in/ns/ext/resize-port#> .
+@prefix unit: <http://lv2plug.in/ns/extensions/units#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/modulay>
+    a lv2:DelayPlugin, lv2:Plugin ;
+
+    lv2:extensionData <http://lv2plug.in/ns/ext/state#interface> ;
+
+    lv2:optionalFeature <http://lv2plug.in/ns/ext/buf-size#boundedBlockLength> ;
+
+    lv2:requiredFeature <http://lv2plug.in/ns/ext/options#options> ,
+                        <http://lv2plug.in/ns/ext/urid#map> ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:AudioPort ;
+        lv2:index 0 ;
+        lv2:symbol "lv2_audio_in_1" ;
+        lv2:name "Audio Input 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 1 ;
+        lv2:symbol "lv2_audio_out_1" ;
+        lv2:name "Audio Output 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 2 ;
+        lv2:name "repeats" ;
+        lv2:symbol "repeats" ;
+        lv2:default 75.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 110.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 3 ;
+        lv2:name "mix" ;
+        lv2:symbol "mix" ;
+        lv2:default 75.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 4 ;
+        lv2:name "rate" ;
+        lv2:symbol "rate" ;
+        lv2:default 2.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 5.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 5 ;
+        lv2:name "depth" ;
+        lv2:symbol "depth" ;
+        lv2:default 1.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 3.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 6 ;
+        lv2:name "time" ;
+        lv2:symbol "time" ;
+        lv2:default 500.000000 ;
+        lv2:minimum 20.000000 ;
+        lv2:maximum 1000.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 7 ;
+        lv2:name "morph" ;
+        lv2:symbol "morph" ;
+        lv2:default 50.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 8 ;
+        lv2:name "tone" ;
+        lv2:symbol "tone" ;
+        lv2:default 3000.000000 ;
+        lv2:minimum 500.000000 ;
+        lv2:maximum 6000.000000 ;
+    ] ;
+
+    rdfs:comment """
+Modulay is a delay with variable types of modulation based on the setting of the Morph-knob.
+""" ;
+
+    mod:brand "SHIRO" ;
+    mod:label "Modulay" ;
+
+    doap:name "Modulay" ;
+    doap:license "ISC" ;
+
+    doap:maintainer [
+        foaf:name "DISTRHO" ;
+        foaf:homepage <https://github.com/DISTRHO/DPF-Max-Gen> ;
+    ] ;
+
+    lv2:microVersion 1 ;
+    lv2:minorVersion 0 .
+
diff --git a/bin/Modulay.lv2/manifest.ttl b/bin/Modulay.lv2/manifest.ttl
new file mode 100644
index 0000000..c96aec2
--- /dev/null
+++ b/bin/Modulay.lv2/manifest.ttl
@@ -0,0 +1,10 @@
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/modulay>
+    a lv2:Plugin ;
+    lv2:binary <Modulay_dsp.so> ;
+    rdfs:seeAlso <Modulay_dsp.ttl> ,
+                 <modgui.ttl> .
+
+
diff --git a/bin/Pitchotto.lv2/Pitchotto_dsp.ttl b/bin/Pitchotto.lv2/Pitchotto_dsp.ttl
new file mode 100644
index 0000000..a215ed2
--- /dev/null
+++ b/bin/Pitchotto.lv2/Pitchotto_dsp.ttl
@@ -0,0 +1,115 @@
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix mod:  <http://moddevices.com/ns/mod#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rsz:  <http://lv2plug.in/ns/ext/resize-port#> .
+@prefix unit: <http://lv2plug.in/ns/extensions/units#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/pitchotto>
+    a lv2:PitchPlugin, lv2:Plugin ;
+
+    lv2:extensionData <http://lv2plug.in/ns/ext/state#interface> ;
+
+    lv2:optionalFeature <http://lv2plug.in/ns/ext/buf-size#boundedBlockLength> ;
+
+    lv2:requiredFeature <http://lv2plug.in/ns/ext/options#options> ,
+                        <http://lv2plug.in/ns/ext/urid#map> ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:AudioPort ;
+        lv2:index 0 ;
+        lv2:symbol "lv2_audio_in_1" ;
+        lv2:name "Audio Input 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 1 ;
+        lv2:symbol "lv2_audio_out_1" ;
+        lv2:name "Audio Output 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 2 ;
+        lv2:name "ratio2" ;
+        lv2:symbol "ratio2" ;
+        lv2:default 2.000000 ;
+        lv2:minimum 0.500000 ;
+        lv2:maximum 2.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 3 ;
+        lv2:name "mix" ;
+        lv2:symbol "mix" ;
+        lv2:default 0.500000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 1.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 4 ;
+        lv2:name "delay1" ;
+        lv2:symbol "delay1" ;
+        lv2:default 100.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 1000.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 5 ;
+        lv2:name "ratio1" ;
+        lv2:symbol "ratio1" ;
+        lv2:default 0.500000 ;
+        lv2:minimum 0.500000 ;
+        lv2:maximum 2.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 6 ;
+        lv2:name "cutoff" ;
+        lv2:symbol "cutoff" ;
+        lv2:default 2250.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 3000.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 7 ;
+        lv2:name "blur" ;
+        lv2:symbol "blur" ;
+        lv2:default 0.250000 ;
+        lv2:minimum 0.010000 ;
+        lv2:maximum 0.250000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 8 ;
+        lv2:name "delay2" ;
+        lv2:symbol "delay2" ;
+        lv2:default 100.000000 ;
+        lv2:minimum 0.100000 ;
+        lv2:maximum 1000.000000 ;
+    ] ;
+
+    rdfs:comment """
+Pitchotto is a pitch-shifter based on the "Pitch-Shift"-genpatch in Max, where Phase-shifting is used to achieve different intervals.
+There are two shifted signals available, both with variable delay-lengths for arpeggio-like sounds.
+""" ;
+
+    mod:brand "SHIRO" ;
+    mod:label "Pitchotto" ;
+
+    doap:name "Pitchotto" ;
+    doap:license "ISC" ;
+
+    doap:maintainer [
+        foaf:name "DISTRHO" ;
+        foaf:homepage <https://github.com/DISTRHO/DPF-Max-Gen> ;
+    ] ;
+
+    lv2:microVersion 1 ;
+    lv2:minorVersion 0 .
+
diff --git a/bin/Pitchotto.lv2/manifest.ttl b/bin/Pitchotto.lv2/manifest.ttl
new file mode 100644
index 0000000..3e55795
--- /dev/null
+++ b/bin/Pitchotto.lv2/manifest.ttl
@@ -0,0 +1,10 @@
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/pitchotto>
+    a lv2:Plugin ;
+    lv2:binary <Pitchotto_dsp.so> ;
+    rdfs:seeAlso <Pitchotto_dsp.ttl> ,
+                 <modgui.ttl> .
+
+
diff --git a/bin/Shiroverb.lv2/Shiroverb_dsp.ttl b/bin/Shiroverb.lv2/Shiroverb_dsp.ttl
new file mode 100644
index 0000000..9891558
--- /dev/null
+++ b/bin/Shiroverb.lv2/Shiroverb_dsp.ttl
@@ -0,0 +1,129 @@
+@prefix doap: <http://usefulinc.com/ns/doap#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix mod:  <http://moddevices.com/ns/mod#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rsz:  <http://lv2plug.in/ns/ext/resize-port#> .
+@prefix unit: <http://lv2plug.in/ns/extensions/units#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/shiroverb>
+    a lv2:ReverbPlugin, lv2:Plugin ;
+
+    lv2:extensionData <http://lv2plug.in/ns/ext/state#interface> ;
+
+    lv2:optionalFeature <http://lv2plug.in/ns/ext/buf-size#boundedBlockLength> ;
+
+    lv2:requiredFeature <http://lv2plug.in/ns/ext/options#options> ,
+                        <http://lv2plug.in/ns/ext/urid#map> ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:AudioPort ;
+        lv2:index 0 ;
+        lv2:symbol "lv2_audio_in_1" ;
+        lv2:name "Audio Input 1" ;
+    ] ;
+
+    lv2:port [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 1 ;
+        lv2:symbol "lv2_audio_out_1" ;
+        lv2:name "Audio Output 1" ;
+    ] ,
+    [
+        a lv2:OutputPort, lv2:AudioPort ;
+        lv2:index 2 ;
+        lv2:symbol "lv2_audio_out_2" ;
+        lv2:name "Audio Output 2" ;
+    ] ;
+
+    lv2:port [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 3 ;
+        lv2:name "shimmer" ;
+        lv2:symbol "shimmer" ;
+        lv2:default 50.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 4 ;
+        lv2:name "decay" ;
+        lv2:symbol "decay" ;
+        lv2:default 50.000000 ;
+        lv2:minimum 1.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 5 ;
+        lv2:name "damping" ;
+        lv2:symbol "damping" ;
+        lv2:default 50.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 6 ;
+        lv2:name "mix" ;
+        lv2:symbol "mix" ;
+        lv2:default 50.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 7 ;
+        lv2:name "ratio" ;
+        lv2:symbol "ratio" ;
+        lv2:default 2.000000 ;
+        lv2:minimum 0.500000 ;
+        lv2:maximum 2.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 8 ;
+        lv2:name "roomsize" ;
+        lv2:symbol "roomsize" ;
+        lv2:default 150.000000 ;
+        lv2:minimum 1.000000 ;
+        lv2:maximum 300.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 9 ;
+        lv2:name "bandwidth" ;
+        lv2:symbol "bandwidth" ;
+        lv2:default 75.000000 ;
+        lv2:minimum 0.000000 ;
+        lv2:maximum 100.000000 ;
+    ] ,
+    [
+        a lv2:InputPort, lv2:ControlPort ;
+        lv2:index 10 ;
+        lv2:name "tone" ;
+        lv2:symbol "tone" ;
+        lv2:default 1500.000000 ;
+        lv2:minimum 500.000000 ;
+        lv2:maximum 6000.000000 ;
+    ] ;
+
+    rdfs:comment """
+Shiroverb is a shimmer-reverb based on the "Gigaverb"-genpatch, ported from the implementation by Juhana Sadeharju, and the "Pitch-Shift"-genpatch, both in Max MSP.
+""" ;
+
+    mod:brand "SHIRO" ;
+    mod:label "Shiroverb" ;
+
+    doap:name "Shiroverb" ;
+    doap:license "ISC" ;
+
+    doap:maintainer [
+        foaf:name "DISTRHO" ;
+        foaf:homepage <https://github.com/DISTRHO/DPF-Max-Gen> ;
+    ] ;
+
+    lv2:microVersion 1 ;
+    lv2:minorVersion 0 .
+
diff --git a/bin/Shiroverb.lv2/manifest.ttl b/bin/Shiroverb.lv2/manifest.ttl
new file mode 100644
index 0000000..fdb1d0a
--- /dev/null
+++ b/bin/Shiroverb.lv2/manifest.ttl
@@ -0,0 +1,10 @@
+@prefix lv2:  <http://lv2plug.in/ns/lv2core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+<https://github.com/ninodewit/SHIRO-Plugins/plugins/shiroverb>
+    a lv2:Plugin ;
+    lv2:binary <Shiroverb_dsp.so> ;
+    rdfs:seeAlso <Shiroverb_dsp.ttl> ,
+                 <modgui.ttl> .
+
+
diff --git a/plugins/Makefile.mk b/plugins/Makefile.mk
index 0eebb9b..7324562 100644
--- a/plugins/Makefile.mk
+++ b/plugins/Makefile.mk
@@ -80,7 +80,7 @@ all:
 
 clean:
 	rm -f *.d *.o
-	rm -rf $(TARGET_DIR)/$(NAME) $(TARGET_DIR)/$(NAME)-* $(TARGET_DIR)/$(NAME).lv2/
+	rm -rf $(TARGET_DIR)/$(NAME) $(TARGET_DIR)/$(NAME)-* $(TARGET_DIR)/$(NAME).lv2/*.so
 
 # --------------------------------------------------------------
 # JACK
-- 
1.9.1

